Libraries/src/Amazon.Lambda.RuntimeSupport/Client/IRuntimeApiClient.cs

```diff
@@ -25,6 +26,11 @@ namespace Amazon.Lambda.RuntimeSupport
     /// </summary>
     public interface IRuntimeApiClient
     {
+        /// <summary>
+        /// Logger used for formatting log messages into the user's CloudWatch Log stream.
+        /// </summary>
+        IConsoleLoggerWriter ConsoleLogger { get; }
```

Comment:
Exposing the logger so I can access it on the bootstrap to record unhandled exceptions with the configured logger..